docs/intro/quickstart/connecting.rst

```diff
@@ -0,0 +1,99 @@
+.. _ref_quickstart_connecting:
+
+==========================
+Connecting to the database
+==========================
+
+.. edb:split-section::
+
+  Before diving into the application, let's take a quick look at how to connect to the database from your code. We will intialize a client and use it to make a simple, static query to the database, and log the result to the console.
+
+  .. note::
+
+    Notice that the ``createClient`` function isn't being passed any connection details. How does |Gel| know how to connect to the database you set up earlier? When we ran ``npx gel project init`` earlier, the CLI created credentials for the local database and stored them in a well-known location. When you initialize your client with ``createClient()``, |Gel| will check the places it knows about for connection details.
+
+    With |Gel|, you do not need to come up with your own scheme for how to build the correct database connection credentials and worry about leaking them into your code. You simply use |Gel| "projects" for local development, and set the appropriate environment variables when you're ready to deploy, and the client knows what to do!
```

Comment:
maybe we include link to the docs with environment variables or connections docs in general

Reply:
Yeah, good idea. Once we have those written, I'll try to circle back and add links.